### DISCUSSION TITLE
GoCardless API Access

### DISCUSSION THREAD

**Discussion by PierfrancescoElia** [firefly-iii/firefly-iii#7646] [Firefly III]
Hi all,
I live in Europe and as far as I know I can only use GoCardless (ex Nordigen) as bank PSD2 intermediate.

Unfortunately I can't understand what type of token I need to obtain. From GoCardless web interface I can request an API key (=alphanumeric string), but on the Data Importer ([here](https://docs.firefly-iii.org/data-importer/how-to-use/select-data-source/)) I can see the Nordigen ID and Nordigen Key fields (2 strings).

It's not clear to me if a Webhook is needed (wouldn't say so) or if I have to become a GoCardless partner.

Can someone explain a more detailed way on how to obtain API ID and Key from GoCardless?

Thanks in advance :)

**Comment by JC5** [comment]
Are you sure you're using the "formerly Nordigen" login at the bottom of the login page?
